### PR TITLE
feat: upgrade generateYMLForNodeProject on http component

### PR DESCRIPTION
**Pull request by zongUMR**
## What has been implemented?

Closes https://app.asana.com/0/1200011502754281/1201300336822086/f

## Steps to verify

- `npm i @serverless/components@canary -g`
- `components init eggjs-starter(express-starter, koa-starter, nextjs-starter, nuxtjs-starter) --name example`
- `cd example`
- ` rm serverless.yml`
- `components deploy`: CLI will automatically generate a new `serverless.yml` for this project via the `http` component and deploy successfully

### CODE REVIEW

**Comment by zongUMR** [comment]
@timqian Please correct me if I am wrong, I removed the last line `throw Error` in the `generate` function, because I think it's impossible to reach out to that error because of the previous checks, and we can finally return the `egg` YML in the end.

File: src/cli/commands-cn/utils.js [hunk 1]
```diff
@@ -536,11 +547,8 @@ inputs:
     return getNuxtYML();
   }
 
-  if (ymlType === 'egg') {
-    return getEggYML();
-  }
-
-  throw new Error('当前目录未检测到 Serverless 配置文件');
+  // finally return egg's yml
+  return getEggYML();
```
Comment (timqian) [inline src/cli/commands-cn/utils.js:551]: Why are we returning YML for egg component here for all cases?

Comment (zongUMR) [reply src/cli/commands-cn/utils.js:551]: We don't return it for all, only for `egg`, due to we have a checker for known packages? https://github.com/serverless/components/pull/1031/files#diff-0e74e48ad16540e1bf681738cd97a41ea2fa6cbdfd5d987b62208fc1a7c829f8R499

Comment (timqian) [reply src/cli/commands-cn/utils.js:551]: Yeah, your are right👍

**Review by timqian — APPROVED** [review]

**Pull request merged**